src/main/java/com/greendays/greendays/service/DailyReportService.java

```diff
@@ -1,8 +1,32 @@
 package com.greendays.greendays.service;
 
+import com.greendays.greendays.model.DailyReport;
+import com.greendays.greendays.repository.ClientRepository;
+import com.greendays.greendays.repository.DailyReportRepository;
+import lombok.RequiredArgsConstructor;
 import org.springframework.stereotype.Service;
 
+import java.sql.Date;
+import java.time.LocalDate;
+import java.time.temporal.TemporalAdjusters;
+import java.util.List;
+
 @Service
+@RequiredArgsConstructor
 public class DailyReportService {
 
+    private final DailyReportRepository dailyReportRepository;
+
+    public String postDailyReport(DailyReport report) {
+        try {
+            dailyReportRepository.save(report);
+        } catch (Exception e) {
+            return e.getMessage();
+        }
+        return "Raport Salvat!";
+    }
+
+    public List<DailyReport> getAllReportsFromThisMonth() {
+        return dailyReportRepository.findAllByDateBetween(Date.valueOf(LocalDate.now().with(TemporalAdjusters.firstDayOfMonth())), Date.valueOf(LocalDate.now().with(TemporalAdjusters.lastDayOfMonth())));
```

Comment:
just realized that if this is the current month, there cannot be a record with the last day of month, maybe us now() instead. we'll fix it later.